NEWS.md

```diff
@@ -22,3 +20,8 @@
 
 #### Posit Workbench
 -
+
+### Updated Dependencies
```

Comment:
Or perhaps just `Dependencies`? Any strong feelings re: word-smithing here?

Reply:
+1 for Dependencies 
sans Updated